flashlight/autograd/Variable.h

```diff
@@ -116,6 +117,43 @@ class Variable {
    */
   af::array& array() const;
 
+  /**
+   * Casts the variable into an appriperiate type based on the value of
```

Comment:
appriperiate ?

Reply:
Thank you, @WilliamTambellini. Fixed. Will be part of the next commit.